kubeflow/trainer/backends/localprocess/job.py

```
@@ -0,0 +1,184 @@
+
+from kubeflow.trainer.constants import constants
+
+logger = logging.getLogger(__name__)
+
+
+class LocalJob(threading.Thread):
+    def __init__(
+        self,
+        name,
+        command: Union[list, tuple[str], str],
+        execution_dir: str = None,
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)